Procurement/ViewModel/ItemHoverViewModel.cs

```diff
@@ -44,6 +44,13 @@ public class ItemHoverViewModel
 
         public string ItemLevel { get; set; }
 
+        public bool IsGemProgressVisible => Item is Gem;
+
+        public double LevelProgress { get; set; }
```

Comment:
Same comments here about the variable names and comments for `LevelProgress`, `Numerator`, and `Denominator`.

Reply:
Made them match up with the variable on the gem.